src/manager.js

```diff
@@ -417,16 +432,16 @@ AdManager.prototype.loadAds = function(element, updateCorrelator) {
  * @returns undefined
 */
 AdManager.prototype.refreshSlot = function(domElement) {
-  if ((domElement.getAttribute('data-ad-load-state') === 'loaded') || (domElement.getAttribute('data-ad-load-state') === 'loading')) {
-    return;
+  var that = this;
+  if (this.options.amazon_enabled) {
```

Comment:
Yeah, that should work.  You could also simply re-initialize the entire script during a `beforeEach` block.  Minor, but we should stick to camelCase for JS variables to stay consistent, so `amazon_enabled` -> `amazonEnabled`

```
adManager = AdManagerWrapper.init({
  dfpSiteCode: 'fmg.onion',
  amazon_enabled: true
});
```